README.md

```diff
@@ -135,8 +134,8 @@ The routes list contains maps, where each object represents a route. For the nex
 
 ## Requirements
 ### Installed Software
-- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
-- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.12.0
+- [Terraform](https://www.terraform.io/downloads.html) ~> 0.12.0
```

Comment:
What do we think about including a line at the top of READMEs for 0.12-requiring modules indicating that there _is_ a tag/branch they can use for <0.12 support? Even better if we link directly to it, rather than making people trawl through the changelog.

Reply:
See https://github.com/terraform-google-modules/terraform-google-network/pull/55